metadetect/metadetect.py

```diff
@@ -92,32 +97,49 @@ class Metadetect(dict):
         Random number generator
     show: bool, optional
         If True, show the images using descwl_coadd.vis.
-    nonshear_mbobs: ngmix.MultiBandObsList, optional
-        If not None athen this mbobs will be sheared and have flux measurements
-        made at the detected positions in the `mbobs`.
+    shear_band_combs: list of list of int, optional
```

Comment:
Why does this need to be a list of lists?

Reply:
Is this for when you want to do bands separately?  In that case each "comb" has a single element?

Reply:
The idea is you pass a list of the sets of bands you want for shear measurement. So for example

```
[[0, 1, 2, 3], [1, 2, 3], [0], [1], [2], [3]]
```

would do a griz, riz, g, r, i, and z shear measurements. Detection happens only with the bands marked for shear, but you get back fluxes for all of the bands.

Reply:
The output data has a new column `shear_bands` that is a string with the band indices used (e.g., "0123", "123", "0", etc.).

Reply:
I think it is of limited  usefulness to have the individual band shears but only detected on that band

Reply:
Ok. So we should change the code to detect on all bands no matter what. will do.